setup.py

```diff
@@ -37,6 +37,7 @@
         "trimesh",
         "httpx",
         "gltflib",
+        "open3d",
```

Comment:
We did some refactoring a few weeks ago, replacing `setup.py` with a `pyproject.toml`. I'm not sure how familar you are with git, but this is what I'd do:

* Undo this change here and commit it.
* Checkout and pull the latest main.
* Then `git merge main` in your branch.
* Add the "open3d" dependency in the right place again.

Reply:
I missed to fetch from upstream before working on the integration, thanks for the hint. I adjusted it and added open3d to the dev requirements.